scripts/register_migging_combiner_analysis.py

```diff
@@ -0,0 +1,38 @@
+"""Register a missing combiner analysis in Metamist
+
+Ad-hoc script to register a missing combiner analysis in Metamist.
+Is supremely inflexible since there is a short timeframe for combining previous
+VDS' for the OurDNA browser release.
+"""
+
+from argparse import ArgumentParser
+
+from hail.vds import read_vds
+
+from cpg_utils import to_path
+from metamist.apis import AnalysisApi
+from metamist.models import Analysis, AnalysisStatus
+
+
+def get_sg_ids(vds: str) -> list[str]:
+    return read_vds(to_path(vds)).variant_data.s.collect()
+
+
+def main(vds: str, dataset: str):
+    aapi = AnalysisApi()
+    am = Analysis(
+        type='combiner',
+        output=vds,
+        status=AnalysisStatus('completed'),
+        sequencing_group_ids=get_sg_ids(vds),
```

Comment:
you'll want to import `cpg_utils.hail_batch.init_batch` and call that, you won't be able to call a Hail method without activating a hail runtime, and that helper method will deal with all the billing project/ref genome stuff

Reply:
I always hecking forget that

Reply:
Alright, that's fixed now